mpop_kiosk/PageSingleQuestion.qml

```diff
@@ -329,7 +343,7 @@ ColumnLayout {
 
                         }
                     }
-                    Button{
+                    Button {
                         text: qsTr("Quebecois")
```

Comment:
@Alex how this text will change

Reply:
There are still a lot of strings to change, but changing it will take some time.

We will need to create an instance of BilingualText for each string, or use if/else conditions.

The PageSingleQuestion item needs a lot of refactoring, so I've opened a refactoring task for that. For this PR, I am not going to do all those changes, because we might very well rewrite 90% of the page, since we want to use more reusable components and a more centralized logic than how it is now. See #270 for details about that. (that backlog item is not ready yet)